graphql_api/types/bundle_analysis/base.py

```diff
@@ -36,6 +39,35 @@ def _find_index_by_cursor(assets: List, cursor: str) -> int:
     return -1
 
 
+def _compute_unknown_asset_size_raw_measurements(fetched_data: dict) -> List[dict]:
+    """
+    Computes measurements for the unknown asset types, some asset types are not in
+    the predetermined list of types so we must compute those measurements manually.
+    The heuristic will be to get the measurements of the bundle type (ie total bundle size)
+    then substract it from all the known asset type measurements, leaving with only the unknown.
+    """
+    unknown_raw_measurements = deepcopy(
+        fetched_data[BundleAnalysisMeasurementsAssetType.REPORT_SIZE][
+            0
+        ].raw_measurements
+    )
+
+    for name, measurements in fetched_data.items():
+        if name not in (
+            BundleAnalysisMeasurementsAssetType.REPORT_SIZE,
+            BundleAnalysisMeasurementsAssetType.ASSET_SIZE,
+        ):
+            raw_measurements = measurements[0].raw_measurements
+            for i in range(len(raw_measurements)):
+                if len(unknown_raw_measurements) != len(raw_measurements):
+                    return []
+                unknown_raw_measurements[i]["min"] -= raw_measurements[i]["min"]
+                unknown_raw_measurements[i]["max"] -= raw_measurements[i]["max"]
+                unknown_raw_measurements[i]["avg"] -= raw_measurements[i]["avg"]
```

Comment:
Yeah true it would suck if the math becomes harder in the future, especially percentage type values.. We might need to rethink an approach to do unknown asset types in the future if it comes to it.